src/supervisor/Supervisor.h

```diff
@@ -143,6 +143,16 @@ class Supervisor : public iosource::IOSource
 	 */
 	struct NodeConfig
 		{
+#pragma GCC diagnostic push
```

Comment:
Maybe just add a comment here about why these are defined and that they can possibly be removed later.

Reply:
Good call Tim, update pushed.